.github/workflows/publish.yaml

```diff
@@ -0,0 +1,22 @@
+name: publish
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+        with:
+          fetch-depth: 1
+      - name: Make
+        run: make all
+      - name: Upload release binaries
+        uses: alexellis/upload-assets@0.1.21
```

Comment:
This version needs to be changed to latest please.

https://github.com/alexellis/upload-assets/releases/